src/Symfony/Component/Config/Util/XmlUtils.php

```diff
@@ -122,7 +122,16 @@ public static function parse($content, $schemaOrCallable = null)
      */
     public static function loadFile($file, $schemaOrCallable = null)
     {
-        $content = @file_get_contents($file);
+        if (!is_file($file)) {
+            throw new \InvalidArgumentException(sprintf('Resource %s is not a file.', $file));
```

Comment:
Can you quote the file name `"%"`? Same below (and if you can also fix the one line 136, that would be awesome).

Reply:
done